mainapp/templates/favlist.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <p>Here's a list of your favorite apartments:</p>
+    {% for apt in favorited_apartments %}
+        <a href=""><h5>{{apt.address}}, {{apt.city}}, {{apt.state}}, {{apt.zipcode}}</h5></a>
```

Comment:
It would be great to have links connected to the associated detail page instead of empty url `""`

Reply:
I agree, if we are going to make each list item a hyperlink, it should link to its detail page

Reply:
Good point. Added hyperlinks to the detailed apartment views.